### PR TITLE
Release 5.2.3

### PR DESCRIPTION
**Pull request by jeeminso**

### CODE REVIEW

**Review by seut — CHANGES_REQUESTED** [review]
Looks good, but please wait for https://github.com/crate/crate/pull/13712 to get merged as it addresses a bug which is marked as fixed inside this release but had some follow up issues.

**Review by seut — APPROVED** [review]
👍

File: docs/appendices/release-notes/5.2.3.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. _version_5.2.3:
+
+=============
+Version 5.2.3
+=============
+
+Released on 2023-02-24.
```
Comment (matriv) [inline docs/appendices/release-notes/5.2.3.rst:7]: Why is it `24` here and not `23` ?

Comment (jeeminso) [reply docs/appendices/release-notes/5.2.3.rst:7]: Just thought 23 end of day and 24 are interchangeable. Fixing this. Thank you !

**Review by matriv — APPROVED** [review]

**Pull request merged**